nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -99,6 +105,16 @@ in {
       '';
     };
 
+    extraInstallCommands = mkOption {
+      default = "";
+      example = "";
+      type = types.lines;
+      description = lib.mdDoc ''
```

Comment:
maybe an example would be nice? just to know for what this could be used

Reply:
I've added an example I was currently playing with. Maybe there is a better way to do that, but I've found that this mechanism that the GRUB module already has can be very helpful when dealing with custom boot entries.